docs/controls/navigation-control.md

```diff
@@ -0,0 +1,25 @@
+# Navigation Control
+
+This is a React equivalent of Mapbox's [NavigationControl](https://www.mapbox.com/mapbox-gl-js/api/#navigationcontrol):
+
+```js
+import ReactMap, {Marker} from 'react-map-gl';
+
+/* Needs Mapbox's stylesheet to work! */
+import 'mapbox-gl/dist/mapbox-gl.css';
```

Comment:
Just realized that this only works in webpack. Our example source should not be bundler dependent.